src/ORM/Connect/NullDatabase.php

```diff
@@ -8,8 +8,7 @@
 /**
  * Utility class required due to bad coupling in framework.
  * Not every framework execution should require a working database connection.
- * For example, when generating class and config manifests for deployment bundles,
- * or when generating code in a silverstripe/graphql schema build.
+ * For example, when generating class and config manifests for deployment bundles
```

Comment:
Still valid - this presumably applies to a front-end schema still.

Reply:
Reverted